src/blocking/spi.rs

```diff
@@ -108,3 +108,134 @@ pub mod write_iter {
         }
     }
 }
+
+/// Provides SpiWithCS wrapper around an spi::* and OutputPin impl
+pub mod spi_with_cs {
+
+    use core::marker::PhantomData;
+    use core::fmt::Debug;
+
+    use crate::digital::OutputPin;
+    use super::*;
```

Comment:
Can we import them explicitly? it is probably just `Transfer`, `Write` and `WriteIter`, right?

Reply:
yep totally